charmcraft/templates/init-go-framework/charmcraft.yaml.j2

```diff
@@ -0,0 +1,55 @@
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+name: {{ name }}
+
+type: charm
+
+base: ubuntu@24.04
+
+platforms:
+  amd64:
+  arm64:
+  armhf:
+  ppc64el:
+  riscv64:
+  s390x:
```

Comment:
OK so I personally love this, but does it accurately represent what's going to work?

While Charmcraft supports building on/for `armhf` and `riscv64`, the Juju snap isn't available for either and IDK whether Juju supports machines running on either. I could potentially see riscv64 as an "upcoming platform", but if there's not a directive to support `armhf` we might want to drop that (or both) for now.

Reply:
Thanks for the info (I just learned that). Dropped both of them.